### DISCUSSION TITLE
New Pattern for a Chicken pattern weight

### DISCUSSION THREAD

**Discussion by darigovresearch** [freesewing/freesewing#5775] [Design request]
Idea from Benjamin F. on the 6th of Jan 2024  community call

https://github.com/freesewing/freesewing/discussions/5532

**Comment by BenJamesBen** [comment]
This is the basic pattern:
https://alexmakesmonsters.com/blogs/news/tutorial-chicken-pattern-weights

- The parts would be the two squares, plus the comb, beak, and tail/loop.
- The default size for the squares would be 5 inches. (In the United States stores commonly sell 5-inch, pre-cut, fabric squares for quilting. These are known as "charm squares".)
- There would be an option to change the size of the squares, with the other parts scaling to match.
- The pattern should calculate the approximate size of the finished chicken and present this information to the customer.

**Comment by BenJamesBen** [comment]
Closing because there does not seem to be any interest for active discussion about this topic, per the April 27, 2024 Community Call. https://github.com/freesewing/freesewing/discussions/5724#discussioncomment-9247197

(Please feel welcome to re-open this Discussion if there is interest in resuming active discussion about the topic.)
